### DISCUSSION TITLE
Zoom

### DISCUSSION THREAD

**Discussion by stoneubi** [TeamNewPipe/NewPipe#10321] [Questions]
Hi there,

I love newpipe and I use it as my main YouTube player.

There are many videos with detailed tutorials on how to e.g. install or maintain a linux system.

What I miss and e.g. YouTube Revanced offers is a zooming function.

I would ask for impending this possibility. 

Thank you
Stoneubi

**Comment by opusforlife2** [comment]
https://github.com/TeamNewPipe/NewPipe/issues/new/choose
